README.md

```diff
@@ -311,6 +311,22 @@ $feedId = $createFeedResult->getFeedId();
 If you are manipulating huge feed documents you can pass to `upload()` anything that Guzzle can turn into a stream.
 
 
+### Uploading with a specific charset
+
+```php
+$charset = "Shift-JIS";
+...
+// Create feed document
+$createFeedDocSpec = new Feeds\CreateFeedDocumentSpecification([
+    'content_type' => SellingPartnerApi\Document::get_content_type_with_charset($feedType['contentType'], $charset)]
```

Comment:
given my below comment, i think this documentation will need to be changed.

i think that a static method for retrieving a content type with a charset should be put in `ContentType.php`, something like this:

```php
public static function withCharset(string $contentType, string $charset): string
{
    return "{$contentType}; charset={$charset}";
}
```